### PR TITLE
ios: i18n

### PR DESCRIPTION
**Pull request by epoberezkin**

### CODE REVIEW

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 1]
```diff
@@ -0,0 +1,796 @@
...
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Your chat profile" xml:space="preserve">
+        <source>Your chat profile</source>
+        <target>Ваш профиль</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Your chat profile will be sent to your contact" xml:space="preserve">
+        <source>Your chat profile will be sent to your contact</source>
+        <target>Ваш профиль будет послан вашему контакту</target>
```
Comment (epoberezkin) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:657]:
```suggestion
        <target>Ваш профиль будет отправлен вашему контакту</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 2]
```diff
@@ -0,0 +1,796 @@
...
+      </trans-unit>
+      <trans-unit id="Connect" xml:space="preserve">
+        <source>Connect</source>
+        <target>Соединиться</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Connect via %@ link?" xml:space="preserve">
+        <source>Connect via %@ link?</source>
+        <target>Соединиться через %@ ссылку?</target>
+        <note>No comment provided by engineer.</note>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:166]:
```suggestion
        <target>Соединиться через ссылку %@?</target>
        <note>Substitution is for "contact" or "invitation" link.</note>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 3]
```diff
@@ -0,0 +1,796 @@
...
+      </trans-unit>
+      <trans-unit id="Contact and all messages will be deleted - this cannot be undone!" xml:space="preserve">
+        <source>Contact and all messages will be deleted - this cannot be undone!</source>
+        <target>Контакт и все сообщения будут удалены - это действие нельзя отменить!</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Contact is connected" xml:space="preserve">
+        <source>Contact is connected</source>
+        <target>Контакт присоединен</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:215]:
```suggestion
        <target>Соединение с контактом установлено</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 4]
```diff
@@ -0,0 +1,796 @@
...
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Edit" xml:space="preserve">
+        <source>Edit</source>
+        <target>Редактировать</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Enter one SMP server per line:" xml:space="preserve">
+        <source>Enter one SMP server per line:</source>
+        <target>Введите SMP серверы, каждый на отдельной строке</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:300]:
```suggestion
        <target>Введите SMP серверы, каждый на отдельной строке:</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 5]
```diff
@@ -0,0 +1,796 @@
...
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Error saving SMP servers" xml:space="preserve">
+        <source>Error saving SMP servers</source>
+        <target>Ошибка при сохранении SMP серверов</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Error: %@" xml:space="preserve">
+        <source>Error: %@</source>
+        <target>Ошибка: </target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:310]:
```suggestion
        <target>Ошибка: %@</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 6]
```diff
@@ -0,0 +1,796 @@
...
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="How to use SimpleX Chat" xml:space="preserve">
+        <source>How to use SimpleX Chat</source>
+        <target>Как использовать SimpleX Chat</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="How to use markdown" xml:space="preserve">
+        <source>How to use markdown</source>
+        <target>Как форматировать</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:345]:
```suggestion
        <target>Форматирование сообщений</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 7]
```diff
@@ -0,0 +1,796 @@
...
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="If you cannot meet in person, you can **scan QR code in the video call**, or your contact can share an invitation link." xml:space="preserve">
+        <source>If you cannot meet in person, you can **scan QR code in the video call**, or your contact can share an invitation link.</source>
+        <target>Если вы не можете встретиться лично, вы можете **сосканировать QR код во время видео-звонка**, или ваш контакт может отправить вам ссылку.</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:350]:
```suggestion
        <target>Если вы не можете встретиться лично, вы можете **сосканировать QR код во время видео звонка**, или ваш контакт может отправить вам ссылку.</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 8]
```diff
@@ -0,0 +1,796 @@
...
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="If you cannot meet in person, you can **show QR code in the video call**, or you can share the invitation link via any other channel." xml:space="preserve">
+        <source>If you cannot meet in person, you can **show QR code in the video call**, or you can share the invitation link via any other channel.</source>
+        <target>Если вы не можете встретиться лично, вы можете **показать QR код во время видео-звонка** или отправить ссылку через любой другой канал связи.</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:355]:
```suggestion
        <target>Если вы не можете встретиться лично, вы можете **показать QR код во время видео звонка** или отправить ссылку через любой другой канал связи.</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 9]
```diff
@@ -0,0 +1,796 @@
...
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="If you received SimpleX Chat invitation link you can open it in your browser:" xml:space="preserve">
+        <source>If you received SimpleX Chat invitation link you can open it in your browser:</source>
+        <target>Если Вы получили ссылку с приглашением из SimpleX Chat, Вы можете открыть её в браузере:</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:360]:
```suggestion
        <target>Если вы получили ссылку с приглашением из SimpleX Chat, вы можете открыть её в браузере:</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 10]
```diff
@@ -0,0 +1,796 @@
...
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Invalid connection link" xml:space="preserve">
+        <source>Invalid connection link</source>
+        <target>Ошибка в ссылке контакта</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Make sure SMP server addresses are in correct format, line separated and are not duplicated." xml:space="preserve">
+        <source>Make sure SMP server addresses are in correct format, line separated and are not duplicated.</source>
+        <target>Пожалуйста, проверьте, что адреса SMP серверов имеют правильный формат, каждый адрес на отдельной строке и не повторяется.</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:375]:
```suggestion
        <target>Пожалуйста проверьте что адреса SMP серверов имеют правильный формат, каждый адрес на отдельной строке и не повторяется.</target>
```

Comment (jr-simplex) [reply apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:375]: I understand lack of punctuation is incorrect but it reads simpler and less official

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 11]
```diff
@@ -0,0 +1,796 @@
...
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Open Settings" xml:space="preserve">
+        <source>Open Settings</source>
+        <target>Открыть Настройки</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Please check that you used the correct link or ask your contact to send you another one." xml:space="preserve">
+        <source>Please check that you used the correct link or ask your contact to send you another one.</source>
+        <target>Пожалуйста проверьте, что вы использовали правильную ссылку или попросите, чтобы ваш контакт отправил вам другую ссылку.</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:415]: for the same reason
```suggestion
        <target>Пожалуйста проверьте что вы использовали правильную ссылку или попросите, чтобы ваш контакт отправил вам другую ссылку.</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 12]
```diff
@@ -0,0 +1,796 @@
...
+      <trans-unit id="Share link" xml:space="preserve">
+        <source>Share link</source>
+        <target>Поделиться ссылкой</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Show QR code to your contact&#10;to scan from the app" xml:space="preserve">
+        <source>Show QR code to your contact
+to scan from the app</source>
+        <target>Покажите QR код Вашему контакту,
+чтобы сосканировать его в приложении</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:507]: you can't say like this in Russian - it means you're the one who'll be doing the scan
```suggestion
        <target>Покажите QR код Вашему контакту для сканирования</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 13]
```diff
@@ -0,0 +1,796 @@
...
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Tap button " xml:space="preserve">
+        <source>Tap button </source>
+        <target>Нажмите кнопку</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Thank you for installing SimpleX Chat!" xml:space="preserve">
+        <source>Thank you for installing SimpleX Chat!</source>
+        <target>Спасибо, что Вы установили SimpleX Chat!</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:527]:
```suggestion
        <target>Спасибо что установили SimpleX Chat!</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 14]
```diff
@@ -0,0 +1,796 @@
...
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="The app can notify you when you receive messages or contact requests - please open settings to enable." xml:space="preserve">
+        <source>The app can notify you when you receive messages or contact requests - please open settings to enable.</source>
+        <target>Приложение может посылать вам уведомления, когда вы получите сообщение или запрос - откройте Настройки чтобы включить.</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:532]:
```suggestion
        <target>Приложение может посылать вам уведомления о сообщениях и запросах на соединение - уведомления можно включить в Настройках.</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 15]
```diff
@@ -0,0 +1,796 @@
...
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="The messaging and application platform 100% private by design!" xml:space="preserve">
+        <source>The messaging and application platform 100% private by design!</source>
+        <target>Платформа для сообщений и приложений, которая защищает вашу личную информацию и безопастность.</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:537]:
```suggestion
        <target>Платформа для сообщений и приложений, которая защищает вашу личную информацию и безопасность.</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 16]
```diff
@@ -0,0 +1,796 @@
...
+        <target>Новый запрос на соединение</target>
+        <note>notification</note>
+      </trans-unit>
+      <trans-unit id="New message" xml:space="preserve">
+        <source>New message</source>
+        <target>Новое сообщение</target>
+        <note>notifications</note>
+      </trans-unit>
+      <trans-unit id="Notification are disabled!" xml:space="preserve">
+        <source>Notification are disabled!</source>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:404]: seems we have a typo in key here, probably easier to fix while there's no localizations

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff (hunk 6 above)
Comment (epoberezkin) [reply apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:345]: Why?

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff (hunk 10 above)
Comment (epoberezkin) [reply apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:375]: 🤷‍♂️

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 17]
```diff
@@ -0,0 +1,796 @@
...
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Trying to connect to the server used to receive messages from this contact (error: %@)." xml:space="preserve">
+        <source>Trying to connect to the server used to receive messages from this contact (error: %@).</source>
+        <target>Устанавливается соединение с сервером, через который Вы получаете сообщения от этого контакта (ошибка: %@).</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Trying to connect to the server used to receive messages from this contact." xml:space="preserve">
+        <source>Trying to connect to the server used to receive messages from this contact.</source>
+        <target>Устанавливается соединение с сервером, через который Вы получаете сообщения от этого контакта.</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:567]:
```suggestion
        <target>Устанавливается соединение с сервером, через который вы получаете сообщения от этого контакта.</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 18]
```diff
@@ -0,0 +1,796 @@
...
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="To start a new chat" xml:space="preserve">
+        <source>To start a new chat</source>
+        <target>Начать новый разговор</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Trying to connect to the server used to receive messages from this contact (error: %@)." xml:space="preserve">
+        <source>Trying to connect to the server used to receive messages from this contact (error: %@).</source>
+        <target>Устанавливается соединение с сервером, через который Вы получаете сообщения от этого контакта (ошибка: %@).</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:562]:
```suggestion
        <target>Устанавливается соединение с сервером, через который вы получаете сообщения от этого контакта (ошибка: %@).</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 19]
```diff
@@ -0,0 +1,796 @@
...
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="You are already connected to %@ via this link." xml:space="preserve">
+        <source>You are already connected to %@ via this link.</source>
+        <target>Вы уже соединены с %@ через эту ссылку.</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="You are connected to the server used to receive messages from this contact." xml:space="preserve">
+        <source>You are connected to the server used to receive messages from this contact.</source>
+        <target>Установлено соединение с сервером, через который Вы получается сообщения от этого контакта.</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:602]:
```suggestion
        <target>Установлено соединение с сервером, через который вы получается сообщения от этого контакта.</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 20]
```diff
@@ -0,0 +1,796 @@
...
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="You can now send messages to %@" xml:space="preserve">
+        <source>You can now send messages to %@</source>
+        <target>Вы теперь можете отправлять сообщения %@</target>
+        <note>notification body</note>
+      </trans-unit>
+      <trans-unit id="You can share your address as a link or as a QR code - anybody will be able to connect to you. You won't lose your contacts if you later delete it." xml:space="preserve">
+        <source>You can share your address as a link or as a QR code - anybody will be able to connect to you. You won't lose your contacts if you later delete it.</source>
+        <target>Вы можете использовать Ваш адрес как ссылку или как QR код - кто угодно сможет соединиться с Вами. Вы сможете удалить адрес, сохранив контакты, которые через него соединились.</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:612]:
```suggestion
        <target>Вы можете использовать ваш адрес как ссылку или как QR код - кто угодно сможет соединиться с вами. Вы сможете удалить адрес, сохранив контакты, которые через него соединились.</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 21]
```diff
@@ -0,0 +1,796 @@
...
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="You can use markdown to format messages:" xml:space="preserve">
+        <source>You can use markdown to format messages:</source>
+        <target>Вы можете форматировать сообщения:</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="You control your chat!" xml:space="preserve">
+        <source>You control your chat!</source>
+        <target>Вы котролируете Ваш чат!</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:622]:
```suggestion
        <target>Вы котролируете ваш чат!</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 22]
```diff
@@ -0,0 +1,796 @@
...
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="You will be connected when your connection request is accepted, please wait or check later!" xml:space="preserve">
+        <source>You will be connected when your connection request is accepted, please wait or check later!</source>
+        <target>Соединение будет установлено, когда Ваш запрос будет принят, пожалуйста подождите или проверьте позже!</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff]:
```suggestion
        <target>Соединение будет установлено, когда ваш запрос будет принят, пожалуйста подождите или проверьте позже!</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 23]
```diff
@@ -0,0 +1,796 @@
...
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="You will be connected when your contact's device is online, please wait or check later!" xml:space="preserve">
+        <source>You will be connected when your contact's device is online, please wait or check later!</source>
+        <target>Соединение будет установлено, когда Ваш контакт будет онлайн, пожалуйста подождите или проверьте позже!</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff]:
```suggestion
        <target>Соединение будет установлено, когда ваш контакт будет онлайн, пожалуйста подождите или проверьте позже!</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 24]
```diff
@@ -0,0 +1,796 @@
...
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Your SMP servers" xml:space="preserve">
+        <source>Your SMP servers</source>
+        <target>Ваши SMP серверы</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:637]:
```suggestion
        <target>SMP серверы</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 25]
```diff
@@ -0,0 +1,796 @@
...
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Your SimpleX contact address" xml:space="preserve">
+        <source>Your SimpleX contact address</source>
+        <target>Ваш SimpleX адрес</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:642]:
```suggestion
        <target>SimpleX адрес</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 26]
```diff
@@ -0,0 +1,796 @@
...
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Your chat address" xml:space="preserve">
+        <source>Your chat address</source>
+        <target>Ваш адрес</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:647]:
```suggestion
        <target>SimpleX адрес</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 27]
```diff
@@ -0,0 +1,796 @@
...
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Your chat profile" xml:space="preserve">
+        <source>Your chat profile</source>
+        <target>Ваш профиль</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:652]:
```suggestion
        <target>Профиль</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 28]
```diff
@@ -0,0 +1,796 @@
...
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Your chat profile will be sent to your contact" xml:space="preserve">
+        <source>Your chat profile will be sent to your contact</source>
+        <target>Ваш профиль будет послан вашему контакту</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Your chats" xml:space="preserve">
+        <source>Your chats</source>
+        <target>Ваши чаты</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:662]:
```suggestion
        <target>Чаты</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 29]
```diff
@@ -0,0 +1,796 @@
...
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Your profile is stored on your device and shared only with your contacts.&#10;SimpleX servers cannot see your profile." xml:space="preserve">
+        <source>Your profile is stored on your device and shared only with your contacts.
+SimpleX servers cannot see your profile.</source>
+        <target>Ваш профиль хранится на вашем устройстве и отправляется только вашим контактам.
+SimpleX серверы не могут получить доступ к Вашему профилю.</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:669]:
```suggestion
SimpleX серверы не могут получить доступ к вашему профилю.</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 30]
```diff
@@ -0,0 +1,796 @@
...
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Your profile will be sent to the contact that you received this link from" xml:space="preserve">
+        <source>Your profile will be sent to the contact that you received this link from</source>
+        <target>Ваш профиль будет отправлен контакту, от которого Вы получили эту ссылку.</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:674]:
```suggestion
        <target>Ваш профиль будет отправлен контакту, от которого вы получили эту ссылку.</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 31]
```diff
@@ -0,0 +1,796 @@
...
+    </header>
+    <body>
+      <trans-unit id="CFBundleName" xml:space="preserve">
+        <source>SimpleX</source>
+        <target>SimpleX</target>
+        <note>Bundle name</note>
+      </trans-unit>
+      <trans-unit id="NSCameraUsageDescription" xml:space="preserve">
+        <source>SimpleX needs camera access to scan QR codes to connect to other app users</source>
+        <target>SimpleX использует камеру для сканирования QR кодов, чтобы соединиться с другими пользователями</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:791]:
```suggestion
        <target>SimpleX использует камеру для сканирования QR кодов при соединении с другими пользователями</target>
```

**Pull request merged**